bindata/assets/kube-controller-manager/svc.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   namespace: openshift-kube-controller-manager
   name: kube-controller-manager
   annotations:
-    service.beta.openshift.io/serving-cert-secret-name: serving-cert
+    service.beta.openshift.io/serving-cert-secret-name: kcm-cpc-reverse-proxy-serving-cert
```

Comment:
we have to change the name of the secret/file because cluster-policy-controller searches this location by default.

But it is causing small hiccups when upgrading. The in between pods crash before the final revision is reached as the moving of the secret is asynchronous.

so we will get a few of these before the final revision is reached

> open /etc/kubernetes/static-pod-resources/secrets/serving-cert/tls.crt: no such file or directory